content/usgs-packages/Application_Intro.Rmd

```diff
@@ -32,14 +32,16 @@ set.seed(1)
 
 ## Lesson Summary
 
-TBD
+This lesson will revisit the packages taught in previous lessons and use some of their functions to create a common data analysis workflow in a modular and reproducible manner. Then, you can learn how a modular approach compares to traditional analyses.
+
+What is modularity and why is it more reproducible? Modularity is the idea that your workflow leverages other tools to do some of the more general tasks. You can think of it as a sort of "plug-and-play" scenario, where you are relying on functionality from another developer to perform pieces of your analysis. This type of workflow reduces some of the maintenance burden because you are pushing some of the responsibility for code accuracy and reliability onto the tools. This also means that you should choose your tools wisely and scrutinize the tools you plan to use in your analyses. 
```

Comment:
Hmmm I think I was trying to allude to  that with talking about reliability and maintenance burden. Now I'm struggling to think of the relationship - does being modular make you more reproducible, or are they two complimentary practices?